src/public/member/verkoperDashboard.php

```diff
@@ -0,0 +1,316 @@
+<style>
+  .table-container {
+    display: flex;
+    justify-content: center;
+    margin-top: 10px;
+    background-color: rgb(64, 64, 64);
+    border-radius: 20px;
+  }
+
+  th,
+  td {
+    border: 12px;
+    padding: 20px;
+    text-align: center;
+  }
+
+  .box {
+    margin-top: 50px; /* mt-12 */
+    border-radius: 20px; /* rounded-2xl */
+    width: auto; /* w-auto */
+    height: auto; /* h-auto */
+    border: 4px solid dimgray; /* border-4 border-solid border-slate-200 */
+    /* Border color is gray */
+    background-color: dimgray; /* bg-slate-200 */
+    /* Background color is gray */
+    padding: 10px; /* p-2.5 */
+    margin: 10px; /* m-2.5 */
+    display: inline-block; /* inline-block */
+  }
+
+  .container {
+    width: 100%;
+    height: 100vh;
+    color: white;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+
+  }
+
+  h2 {
+    font-size: 2rem;
+    margin-bottom: 1rem;
+    text-align: center;
+  }
+
+  .description {
+    text-align: center;
+    width: 43%;
+  }
+
+  .clientImage {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .clientImage span {
+    margin-left: 10px;
+  }
+
+  .clientImage img {
+    width: 40px;
+  }
+
+  .reviewSection {
+    padding: 1rem;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-around;
+  }
+
+  .reviewItem {
+    width: 300px;
+    padding: 10px;
+    margin: 1rem;
+    cursor: pointer;
+    border-radius: 10px;
+
+    border: 1px solid #10102a;
+    transition: all .2s linear;
+  }
+
+  .reviewItem:hover {
+    border-color: aqua;
+    transform: scale(1.01);
+
+    box-shadow: 0 0px 5px 0px #cbc0c0;
+  }
+
+  .top {
+    margin-bottom: 1rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .top ul {
+    display: flex;
+    list-style: none;
+  }
+
+  .top ul li {
+    padding-left: 4px;
+  }
+
+  article p {
+    font-size: 15px;
+    font-weight: 100;
+    margin-bottom: 1rem;
+    font-family: system-ui;
+  }
+
+
+  @media screen and (max-width:700px) {
+    .container {
+      height: auto;
+    }
+
+    .description {
+      width: 90%;
+    }
+  }
+
+  @media screen and (max-width:375px) {
+    .reviewSection {
+      padding: 0;
+    }
+
+    .reviewItem {
+      width: 100%;
+    }
+
+    .clientImage {
+      margin-bottom: 0.6rem;
+    }
+
+    .top {
+      align-items: center;
+      flex-direction: column;
+      justify-content: center;
+    }
+  }
+</style>
```

Comment:
Replace all these css styles with tailwind classes (https://tailwindcss.com/docs/padding), just look up the css properties used in the file, on the tailwind website and use those as classes for the corresponding html tags.

I've written comments behind the css properties of the box class as an example, those classes also have to be added to all divs/elements that have the 'box' class

Reply:
I've tried changing the css properties to tailwind classes, but it changes from drastically to barely noticeable parts of the design.

Reply:
mhh alright, I'll take a look.